airflow/cli/cli_parser.py

```diff
@@ -245,6 +245,9 @@ def string_lower_type(val):
     choices={ColorMode.ON, ColorMode.OFF, ColorMode.AUTO},
     default=ColorMode.AUTO,
 )
+ARG_DAG_ID_REQ_FLAG = Arg(
+    ("-d", "--dag-id"), required=True, help="The id of the dag"
```

Comment:
```suggestion
    ("-d", "--dag-id"), required=True, help="The ID of the dag"
```

Reply:
I'd rather leave this as-is in this pr, matching the existing help text for the other dag id arg.